validate/validate.go

```diff
@@ -423,6 +423,10 @@ func (v *Validator) CheckCapabilities() (errs error) {
 
 // CheckRlimits checks v.spec.Process.Rlimits
 func (v *Validator) CheckRlimits() (errs error) {
+	if v.platform == "windows" {
```

Comment:
Can you add a warning output on this side? such as:
>logrus.Warnf("process.rlimits validation not yet implemented for platform %q", v.platform)

Also, does this have some overlap with the validation of the platform in `rlimitValid`?

Reply:
> Can you add a warning output on this side?

I don't think we want that.  “Not yet implemented” warnings are for cases where runtime-tools has not yet caught up with the spec's requirements.  But [Windows does not, and likely never will support `rlimits`][1] and [the spec does not define `rlimits` for Windows][2].

> Also, does this have some overlap with the validation of the platform in `rlimitValid`?

I'm touching `rlimitValid` in this PR.  Do you think we need `rlimitValid` changes beyond those?

[1]: https://github.com/opencontainers/runtime-spec/pull/835#issuecomment-303455386
[2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#posix-process